shared/actions/platform-specific/index.native.js

```diff
@@ -213,7 +220,11 @@ function* loadStartupDetails() {
   let startupLink = ''
   let startupTab = null
 
-  const routeStateTask = yield Saga.fork(AsyncStorage.getItem, 'routeState')
+  const routeStateTask = yield Saga.fork(() =>
```

Comment:
the previous code figures this all out in one shot very early so i had to change how we init sagas (below)